apps/webapp/src/components/header/header.tsx

```diff
@@ -1,11 +1,21 @@
-import { Navbar } from './navbar';
+import { LayoutLoaderResult } from '../layout.tsx';
 import { Link, useLoaderData } from 'react-router-dom';
-import { PagePath } from '../metadata/paths.ts';
+import { MessageWarningIcon } from '../../icons/message-warning.tsx';
+import { MobileNavMenu } from './mobile-nav-menu.tsx';
+import { Navbar } from './navbar';
+import {
+  NetworksPopover,
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from '@penumbra-zone/ui';
 import Notifications from './notifications.tsx';
+import { PagePath } from '../metadata/paths.ts';
 import { TabletNavMenu } from './tablet-nav-menu.tsx';
-import { LayoutLoaderResult } from '../layout.tsx';
-import { NetworksPopover } from '@penumbra-zone/ui';
-import { MobileNavMenu } from './mobile-nav-menu.tsx';
+
+const WEB_EXT_FEEDBACK_DISCORD_CHANNEL =
+  'https://discord.com/channels/824484045370818580/1077672871251415141';
```

Comment:
Note that this is just the public link of the channel, which I got by right-clicking the channel and choosing "Copy link":
![image](https://github.com/penumbra-zone/web/assets/1121544/a9a6459e-8bf8-4735-959f-097ec3a60a31)

If it should be an invite link rather than the public link, someone other than me should create the invite link, since I'm only a contractor.

Reply:
Try https://discord.gg/XDNcrhKVwV

Reply:
That link should be an infinite-expiry invite to the channel

Reply:
Ah, thanks @hdevalence !